src/components/Common/AuthorizedButton.tsx

```diff
@@ -1,12 +1,15 @@
-import { Button, ButtonProps } from "@headlessui/react";
+import * as React from "react";
 
 import AuthorizedChild from "@/CAREUI/misc/AuthorizedChild";
 
 import { AuthorizedElementProps } from "@/Utils/AuthorizeFor";
 
-export const AuthorizedButton: React.FC<
-  AuthorizedElementProps & ButtonProps
-> = ({ authorizeFor = () => true, ...props }) => {
+import { Button, ButtonProps } from "../ui/button";
+
+const AuthorizedButton: React.FC<AuthorizedElementProps & ButtonProps> = ({
+  authorizeFor = () => true,
+  ...props
+}) => {
```

Comment:
let's drop the default. If we are going to use `AuthorizedButton`, then let's mandate the prop. If we are not going to pass the authorizeFor prop, we should be using the normal Button itself.

```suggestion
const AuthorizedButton: React.FC<AuthorizedElementProps & ButtonProps> = (props) => {
```

Reply:
@rithviknishad can you once check